packages/richtext/package.json

```diff
@@ -0,0 +1,87 @@
+{
+    "name": "@sitecore-content-sdk/richtext",
+    "version": "0.2.0-canary.18",
+    "main": "dist/cjs/index.js",
+    "module": "dist/esm/index.js",
+    "sideEffects": false,
+    "publishConfig": {
+        "access": "public",
+        "registry": "https://registry.npmjs.org/"
+    },
+    "scripts": {
+        "build": "npm run clean && tsc -p tsconfig.json && tsc -p tsconfig-esm.json",
+        "clean": "del-cli dist types",
+        "lint": "eslint \"./src/**/*.ts\"",
+        "test": "mocha \"./src/**/*.test.ts\" --exit",
+        "prepublishOnly": "npm run build",
+        "coverage": "nyc npm test",
+        "generate-docs": "npx typedoc --plugin typedoc-plugin-markdown --outputFileStrategy Members --parametersFormat table --readme none --githubPages false"
+    },
+    "engines": {
+        "node": ">=22"
+    },
+    "author": {
+        "name": "Sitecore Corporation",
+        "url": "https://doc.sitecore.com/xmc/en/developers/content-sdk/index.html"
+    },
+    "license": "Apache-2.0",
+    "homepage": "https://doc.sitecore.com/xmc/en/developers/content-sdk/index.html",
+    "bugs": {
+        "url": "https://github.com/sitecore/content-sdk/issues"
+    },
+    "devDependencies": {
+        "@types/chai": "^4.3.4",
+        "@types/chai-as-promised": "^7.1.5",
+        "@types/chai-string": "^1.4.2",
+        "@types/mocha": "^10.0.1",
+        "@types/node": "~22.9.0",
+        "@types/sinon": "^10.0.13",
+        "@types/sinon-chai": "^3.2.9",
+        "chai": "^4.3.7",
+        "chai-as-promised": "^7.1.1",
+        "chai-string": "^1.5.0",
+        "chalk": "^4.1.2",
+        "del-cli": "^6.0.0",
+        "eslint": "^8.56.0",
+        "eslint-plugin-react": "^7.32.1",
+        "jsdom": "^26.0.0",
+        "mocha": "^11.1.0",
+        "nyc": "^17.1.0",
+        "sinon": "^19.0.2",
+        "sinon-chai": "^3.7.0",
+        "tsx": "^4.19.2",
+        "typescript": "~5.7.3"
+    },
+    "peerDependencies": {
+        "@tiptap/core": "^2.11.7",
+        "@tiptap/html": "^2.11.7",
+        "@tiptap/starter-kit": "^2.11.7",
+        "typescript": "~5.7.3"
+    },
+    "peerDependenciesMeta": {
+        "@tiptap/core": {
```

Comment:
what will happen it user installs "richtext" package, but skip installing tiptap, and call `parseTiptapJSON`

Reply:
I'll check, probably need to remove Meta here so that peers install in any case

Reply:
Works fine with Meta and without - so I'll remove it. This is a monorepo setup of course, the standalone app will need tiptap peer dependencies installed to ensure package and the app are on the same page.